resolver/resolver.cc

```diff
@@ -1099,9 +1099,7 @@ class ResolveTypeMembersWalk {
             return;
         }
 
-        // NOTE: AttachedClass will not exist on `T.untyped`, which is a problem
-        // because RuntimeProfiled is used as a synonym for `T.untyped`
-        // internally.
+        // NOTE: AttachedClass will not exist on `T.untyped`.
         auto attachedClass = singleton.data(ctx)->findMember(ctx, core::Names::Constants::AttachedClass());
         if (!attachedClass.exists()) {
```

Comment:
It's not obvious to me what to do here (or the related, but currently untouched by this PR) code in `resolver/GlobalPass.cc`.  Guidance appreciated, or "it is reasonable to punt this surgery to a future PR".

Reply:
cc @elliottt do you remember what this comment means?

My hunch is that it means we can ENFORCE that this exists now?

Reply:
At the very least can you leave the original comment so that we can still have the context when looking at this code in the future?

Maybe someone will say "but wait we don't have RuntimeProfiled anymore" and then go figure out what's best.

Reply:
https://github.com/sorbet/sorbet/blob/9a85753fab0a03e9d43cc676e8b3da14a059af9d/gems/sorbet-runtime/lib/types/types/attached_class.rb#L5-L8 suggests `ENFORCE`'ing `attachedClass.exists()` is correct.

Reply:
if you want to make the change and see whether tests pass, 👍

Reply:
Blah, building sorbet failed while trying to regenerate the payload.  I'm going to put the comment back the way it was and leave that investigation for another day.